.github/renovate.json5

```diff
@@ -6,21 +6,19 @@
     ":disableDependencyDashboard"
   ],
   "enabledManagers": [
-    // Enable only the regex manager (for Dockerfile base image bumping). Go dependencies are managed by Dependabot.
+    // Enable only the regex manager. Go dependencies are managed by Dependabot.
     "regex",
     // Managers for helm and helm-values. Go dependencies are managed by Dependabot.
-    "helm-values",
     "helmv3"
   ],
   "regexManagers": [
     {
-      // Parse bundle image version from `BASE_IMAGE` ARG in Dockerfile.
       "fileMatch": [
-        "^Dockerfile$"
```

Comment:
This seems to be a leftover coping from `nri-ecs`: https://github.com/newrelic/nri-ecs/blob/main/.github/renovate.json5

The previous configuration was useless in this repository.

Thie comment also applies to the change below.